recipe/meta.yaml

```diff
@@ -1,49 +1,68 @@
 {% set version = "3.18.0" %}
-
+{% set name = 'freeimage' %}
 
 package:
-    name: freeimage
+    name: {{ name }}
     version: {{ version }}
-    # note: also update version in run_test.py
+    # note: also update version in CMakeLists and in tests.
 
 source:
-    fn: FreeImage{{ version.replace(".", "") }}.zip
-    url: http://downloads.sourceforge.net/freeimage/FreeImage{{ version.replace(".", "") }}.zip
+  - url: http://downloads.sourceforge.net/freeimage/FreeImage{{ version.replace('.', '') }}.zip
     sha256: f41379682f9ada94ea7b34fe86bf9ee00935a3147be41b6569c9605a53e438fd
-    patches:
-        - Makefile.osx.patch  # [osx]
+
+  - path: CMakeLists.txt
```

Comment:
What is the source of these files? Written for this recipe or taken from somewhere else?

Reply:
inspired from VCPKG :)

Reply:
do you want me to add a note?

Reply:
Would be nice.